Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+// Obtain files from source control system.
+if (utils.scm_checkout()) return
+
+// Globals
+PIP_INST = "pip install"
+CONDA_INST = "conda install -y -q"
+PY_SETUP = "python setup.py"
+PYTEST_ARGS = "--basetemp=tests_output --junitxml results.xml"
```

Comment:
How do you feel about adding `--capture=sys` or `--capture=fd` to `PYTEST_ARGS`? I'm noticing that IRAF tasks are inconsistently verbose, and PyRAF doesn't always redirect `stdout` and `stderr` streams correctly when instructed to.

Reply:
Why not? Let's do it. It's a wild wild West here.

Reply:
@jhunkeler , wait a minute. Now I remember that I already have `addopts = --capture=no` in `setup.cfg`. Is that not sufficient? Won't Jenkins be smart enough to pick up `pytest` options from there?